pkg/controllers/default-resources.go

```diff
@@ -28,12 +28,20 @@ import (
 
 const ReleaseImage = "quay.io/openshift-release-dev/ocp-release:4.9.15-x86_64"
 
+func getTargetNamespace(hyd *hypdeployment.HypershiftDeployment) string {
+	if len(hyd.Spec.TargetNamespace) == 0 {
+		return hyd.GetNamespace()
```

Comment:
Maybe we log a message about the target namespace, since this is called by scaffold, it only runs once